app/schemas/models/user.coffee

```diff
@@ -346,6 +346,16 @@ _.extend UserSchema.properties,
   role: {type: 'string', enum: ["God", "advisor", "parent", "principal", "student", "superintendent", "teacher", "technology coordinator"]}
   birthday: c.stringDate({title: "Birthday"})
   lastAchievementChecked: c.stringDate({ name: 'Last Achievement Checked' })
+  signupQueryParams: { type: 'object', properties: {
+    il_id: {type: 'string'}
+    first_name: {type: 'string'}
```

Comment:
Any particular reason to redundantly store this along with `firstName`? (same for `lastName`, `email`). Just to provide more context for the info?
Also, any reason these are `snake_case` instead of `camelCase` like the top-level user attributes?